test/functional/testlib.bash

```diff
@@ -2259,15 +2261,16 @@ generate_test() {
 		printf 'global_teardown() {\n\n'
 		printf '\t# global cleanup\n\n'
 		printf '}\n\n'
-		printf '@test "<test description>" {\n\n'
+		printf '@test "<test ID>: <test description>" {\n\n'
```

Comment:
I am concerned that it would be annoying to figure out the next available test ID value when adding a new test. Could this function populate the test ID with an open number? Maybe a grep command could be used to read existing IDs and help pick an available one.

Reply:
For now we can leave like this and plan to have a script to create IDs later. Let's see how this work first. If it's getting complicated we can write a better solution

Reply:
Ok